.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: Using MetaMask, but it's not working as you expect?
+
+---
+
+<!--
+BEFORE SUBMITTING: PLEASE SEARCH TO MAKE SURE THIS ISSUE HAS NOT BEEN SUBMITTED
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Browser details (please complete the following information):**
```

Comment:
can we add a metamask details here that asks for current installed version of mm and if they're using the old or new ui?

Reply:
yes good call